docs/conversion.md

```diff
@@ -33,7 +33,7 @@ __Glossary:__
 | deploy: replicas       | -  | -  | ✓  | Deployment.Spec.Replicas / DeploymentConfig.Spec.Replicas   |                                                                                                                |
 | deploy: placement      | -  | -  | ✓  | Pod.Spec.NodeSelector                                       |                                                                                                                |
 | deploy: update_config  | -  | -  | n  |                                                             |                                                                                                                |
-| deploy: resources      | -  | -  | ✓  | Containers.Resources.Limits.Memory                          | Support for memory but not CPU                                                                                 |
+| deploy: resources      | -  | -  | ✓  | Containers.Resources.Limits.Memory and cpu                  | Support for memory as well as cpu                                                                    |
```

Comment:
Keep to the format of the other keys...

Just change this to `Containers.Resources.Limits.Memory / Containers.Resources.Limits.CPU`